src/clients/billingClient.ts

```diff
@@ -23,6 +23,9 @@ export default class Billing {
       data: { data, errors },
     } = await this.http.postRaw<any>(`/_v/graphql`, { query: graphQLQuery })
     if (errors) {
+      if (errors.length === 1 && errors[0].extensions?.exception?.response?.data) {
+        throw errors[0].extensions.exception.response.data
+      }
```

Comment:
Maybe we could add this to `ErrorReport` as well? Or check how ErrorReport interacts with graphqlErrors and try to improve?